docs/4. API/Server/Session-based verification/7-create2-verification-compile.mdx

```diff
@@ -0,0 +1,57 @@
+---
+id: session-create2-compile
+title: Create2 Compilation
+slug: /api/server/session/verify/create2/compile
+---
+
+import V2Note from "./_v2Note.mdx";
+
+# Create2 Compilation
+
+In order to verify using the create2 api, the creation bytecode is needed. With this API the server compiles the contract and puts the creation bytecode in the session.
```

Comment:
So does this endpoint need to be called before `/session/verify/create2` ? If so we should indicate that

Reply:
What do you mean specify? 

like instead of this:

> In order to verify using the create2 api

this?

> In order to verify using `/session/verify/create2`

Reply:
No I meant if you want to call `/session/verify/create2` you need to call `/session/verify/create2/compile` right? We do that automatically on the UI but otherwise, this needs to be done in another way